db/schema.rb

```diff
@@ -26,6 +26,17 @@
     t.index ["shelter_id"], name: "index_pets_on_shelter_id"
   end
 
+  create_table "reviews", force: :cascade do |t|
+    t.string "title"
+    t.integer "rating"
+    t.string "content"
+    t.string "picture"
```

Comment:
one thing we have to do is add the picture as an "optional picture" by giving it a default